adding extensionAttributes for devices (#8389)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/device.md

|deviceMetadata|String| For internal use only. Set to null. |
|deviceVersion|Int32| For internal use only. |
|displayName|String| The display name for the device. Required. |

> | extensionAttributes | [extensionAttributes](extensionattributes.md) | Contains extensionAttributes 1-15 for the device. The individual extension attributes are neither selectable nor filterable. This set of properties is mastered in cloud and may be set during creation or update of a device. <br><br>Returned only on $select. |

Reviewer: Not sure the filterable is correct now.  I believe that Mezzo now allows filtering on extensionAttribute individual items - at least it works on user.

> | extensionAttributes | [extensionAttributes](extensionattributes.md) | Contains extensionAttributes 1-15 for the device. The individual extension attributes are neither selectable nor filterable. This set of properties is mastered in cloud and may be set during creation or update of a device. <br><br>Returned only on $select. |

Reviewer 2: ```suggestion
| extensionAttributes | [extensionAttributes](extensionattributes.md) | Contains extensionAttributes 1-15 for the device. The individual extension attributes are not selectable or filterable. This set of properties may be set during creation or update of a device. <br><br>Returned only on $select. |
```

> | extensionAttributes | [extensionAttributes](extensionattributes.md) | Contains extensionAttributes 1-15 for the device. Note that the individual extension attributes are neither selectable nor filterable. This set of properties is mastered in cloud and may be set during creation or update of a device. <br><br>Returned only on $select. |

Reviewer 2: Contains extension attributes 1-15 for the device. The individual extension attributes are not selectable or filterable. This set of properties is mastered in the cloud and may be set during creation or update of a device. Returned only on $select.

Author: Will do
